desktop_version/src/Game.h

```diff
@@ -10,7 +10,7 @@
 
 struct MenuOption
 {
-    std::string text;
+    char text[41]; // 40 chars covers the entire screen, + 1 more for null terminator
```

Comment:
You're overlooking UTF-8 not using single bytes per character here, so the buffers would need to be 161 chars to fit the longest possible visible string, especially when menus are translated (ditto the other temporary strings).

Reply:
Fixed, rebased and force-pushed.

Reply:
Why not simply add an extra commit instead of rebasing and force pushing, if I may ask? Your commit description is now inconsistent with the action taken in that commit.

Reply:
Because I like having a clean commit history. [Here's an example of what I want to avoid](https://github.com/FNA-XNA/FNA/pull/280#issuecomment-580060023).

Nevertheless, I've rebased and force-pushed to correct that commit description.